x-pack/plugins/alerting/server/routes/rule/apis/create/create_rule_route.ts

```diff
@@ -37,8 +37,15 @@ export const createRuleRoute = ({ router, licenseState, usageCounter }: RouteOpt
         summary: `Create a rule`,
       },
       validate: {
-        body: createBodySchemaV1,
-        params: createParamsSchemaV1,
+        request: {
+          body: createBodySchemaV1,
+          params: createParamsSchemaV1,
+        },
+        response: {
+          200: {
+            body: () => ruleResponseSchema,
```

Comment:
I'm a noob looking into this 😅  How does this work, why `body: () => ruleResponseSchema` instead of `body: ruleResponseSchema`?

Reply:
As per `packages/core/http/core-http-server/src/router/route_validator.ts`, it expects `() => Type<unknown>`.